pipeline/notifications/rating_reminder.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from pipeline.notifications.notifier import Notifier
+from pipeline.types import NotificationInfo, NotificationInput, NotificationType
+
+
+class RatingReminderNotifier(Notifier):
+    def __init__(self):
+        pass
+
+    def get_notifications(
+        self, inp: NotificationInput
+    ) -> List[NotificationInfo]:
+        remind_users = []
+        min_chat: int = 3
```

Comment:
No need to change this now, but for future cases, something like this could be a parameter set in the class constructor, since it is a configurable value.